Extension/package.json

```diff
@@ -39,7 +39,31 @@
     "Snippets"
   ],
   "activationEvents": [
-    "*"
+    "onLanguage:cpp",
```

Comment:
This activationEvent change should be reverted (manually is fine). It's an artifact of the fact that we modify the package.json on runtime, i.e. the normal procedure to update our package.json would be to undo this change first, apply the change, stage/commit the change, then Launch extension (which would change the activationEvents again). The other alternative is to copy the Extension folder to a different location to use for debugging.